python/tests/datasources/test_graphql_ahr.py

```diff
@@ -11,18 +11,16 @@
 TEST_DIR = os.path.join(THIS_DIR, os.pardir, "data", "graphql_ahr")
 GOLDEN_DIR = os.path.join(TEST_DIR, "golden_data")
 
-EXPECTED_AHR_API_RESPONSE_DATA = os.path.join(TEST_DIR, 'response_data.json')
-
 GOLDEN_DATA = {
     'age_national_current': os.path.join(GOLDEN_DIR, 'age_national_current.csv'),
     'sex_national_current': os.path.join(GOLDEN_DIR, 'sex_national_current.csv'),
     'race_and_ethnicity_state_current': os.path.join(GOLDEN_DIR, 'race_and_ethnicity_state_current.csv'),
 }
 
 
-def _fetch_ahr_data_from_graphql():
+def _fetch_ahr_data_from_graphql(demographic: str, geo_level: str):
     print("MOCK - AHR GraphQL API response")
```

Comment:
might be nice to update this to log out the mocked demographic/geo_level being loaded; definitely might help future debugging